ansible-ipi-install/inventory/hosts.sample

```diff
@@ -56,15 +56,17 @@ pullsecret=""
 
 
 # Master nodes
+# The hardware_profile is used by the baremetal operator to match the hardware discovered on the host
+# See https://github.com/metal3-io/baremetal-operator/blob/master/docs/api.md for more information
```

Comment:
Can we use this link instead? https://github.com/metal3-io/baremetal-operator/blob/master/docs/api.md#baremetalhost-status